src/odemis/driver/scanner.py

```diff
@@ -130,7 +132,7 @@ def __init__(self, name, role, children=None, dependencies=None, daemon=None, **
             else:
                 logging.debug("No external VA supported for the Composited Scanner.")
 
-        else:
+        if not self.children.value:  # If no automatic blanker/external, just duplicate the original ones
```

Comment:
This comment is not very fitting for this line could you move this just above (or behind) this line `va_names += ["blanker", "external"]`